CHANGELOG.md

```diff
@@ -2,14 +2,19 @@
 
 <!-- prettier-ignore-start -->
 > [!IMPORTANT]
-> If you are upgrading to the `1.x` versions of the SDK from `0.x`, make sure you follow our
+> If you are upgrading to the `2.x` versions of the SDK from `1.x` or lower, make sure you follow our
 > [migration guide](https://docs.sentry.io/platforms/javascript/guides/capacitor/migration/) first.
 <!-- prettier-ignore-end -->
 
 ## Unreleased
 
 ### Dependencies
 
+- Bump JavaScript SDKs from v8.55.0 to v9.0.0 ([#840](https://github.com/getsentry/sentry-capacitor/pull/840))
+
+  - [changelog](https://github.com/getsentry/sentry-javascript/blob/9.0.0/CHANGELOG.md#900)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/8.55.0...9.0.0)
+
 - Bump Sentry Javascript to 8.55.0 ([#870](https://github.com/getsentry/sentry-capacitor/pull/870))
```

Comment:
Let's release the 8.55.0 bump first, and then the v9.

Reply:
8.55.0 was already releasem i fixed the merge conflct here https://github.com/getsentry/sentry-capacitor/pull/840/commits/2f7bd2916a7509cb5863bf76af581a6ae54e1c96